frontend/src/pages/detail/ModfiyBook.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { FormEvent } from "react";
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { ToggleButton } from "primereact/togglebutton";
+import ConfirmButton from "../../components/ConfirmButton";
+
+interface modifyBookState {
+  title: string;
```

Comment:
Rather than explicitly defining all of these strings, I think it would be better if "BookRow" was exported from BookList (and the name can be changed to "Book" as well) and we just had "book: BookRow" as the state. That way, whenever we modify the BookRow object (to maybe include more fields), it is automatically reflected here

Reply:
And again, change the "checked" and "confirmationPopup" booleans to be more descriptive